core/services/vrf/listener_v2.go

```diff
@@ -374,6 +374,8 @@ func (lsn *listenerV2) processPendingVRFRequests(ctx context.Context) {
 								lsn.job.VRFSpec.BackoffMaxDelay,
 								req.lastTry))
 					}
+				} else {
+					lsn.markLogAsConsumed(req.lb)
```

Comment:
Now that we have this here, can we remove all other instances of lsn.markLogAsConsumed(req.lb)?

Reply:
Good suggestion. We can remove the other `markLogAsConsumed` calls while processing VRF requests. In `handleLogs` method, which is a separate listener that filters out fulfilled logs and parse failures, we should keep `markLogAsConsumed`

Reply:
Ah yeah good catch, still need that one. Sweet